extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/HibernateValidatorTemplate.java

```diff
@@ -45,14 +51,17 @@ public void initializeValidatorFactory(Set<Class<?>> classesToBeValidated) {
             method.invoke(null);
         } catch (Throwable t) {
             //if EL is not on the class path we use the parameter message interpolator
-            configuration.messageInterpolator(new ParameterMessageInterpolator(localesToInitialize));
+            ParameterMessageInterpolator parameterMessageInterpolator = new ParameterMessageInterpolator(localesToInitialize);
+            predefinedScopeValidatorConfiguration.messageInterpolator(parameterMessageInterpolator);
+            hibernateValidatorConfiguration.messageInterpolator(parameterMessageInterpolator);
         }
 
-        configuration
+        predefinedScopeValidatorConfiguration
                 .initializeBeanMetaData(classesToBeValidated)
                 .initializeLocales(localesToInitialize)
                 .beanMetaDataClassNormalizer(new ArcProxyBeanMetaDataClassNormalizer());
 
-        ValidatorHolder.initialize(configuration.buildValidatorFactory());
+        ValidatorHolder.initializeHibernateValidator(hibernateValidatorConfiguration.buildValidatorFactory());
+        ValidatorHolder.initializePredefinedScopeValidator(predefinedScopeValidatorConfiguration.buildValidatorFactory());
```

Comment:
I don't understand why you define 2 different validators?

Reply:
It is because I was getting this 
```
java.lang.IllegalStateException: Defining a Validator-specific traversable resolver is not supported by the predefined scope ValidatorFactory.
```

when trying to use `PredefinedScopeHibernateValidatorFactory`. 

Any ideas?

Reply:
Hmmm, right. So that's something we need to fix in HV. I was a bit too conservative on what would be accepted.

TBH, I didn't think the traversable resolver would be set for each JPA validation call.

Could you change your code to use the initial `ValidatorFactory` (well, it will mostly be a matter of reverting changes)? I'll push a PR to HV so that you can test if it works OK for you.

Reply:
@machi1990 can you check with this branch of Hibernate Validator: https://github.com/hibernate/hibernate-validator/pull/1039 ?

It should solve the issue and you should be able to have only one VF.

I'll wait for your feedback before merging and releasing a new alpha.

Reply:
> @machi1990 can you check with this branch of Hibernate Validator: [hibernate/hibernate-validator#1039](https://github.com/hibernate/hibernate-validator/pull/1039) ?
> 
> It should solve the issue and you should be able to have only one VF.
> 
> I'll wait for your feedback before merging and releasing a new alpha.

@gsmet thanks for the quick response. It solves the problem.

Reply:
@gsmet Hi, any news on the HV release?

Reply:
Sorry, still in the pipeline. I'm preparing a talk and didn't find the time to do it.

I think I should be able to push the release button this afternoon.

Reply:
Cool :-)

Reply:
OK, release done. Now, we have to wait for it to be synced to Central. Hopefully, it will be synced today and I'll push an upgrade PR to fix another issue and prepare for this one.